### PR TITLE
add text explaining why we want profile info, fixes #565

### LINKED ISSUE #565: Why Profile Text
Assigning this to you @mikehenrty, because I'm not sure if you'd like my prose for this
https://github.com/mozilla/voice-web/blob/c4202c479b1caa9014f731e6d8af35f0ff15f68a/web/src/lib/components/pages/record/record.tsx#L454-L456

### PR DESCRIPTION
**Pull request by mikehenrty**

### CODE REVIEW

**Comment by mikehenrty** [comment]
@kdavis-mozilla Kelly, I took a first pass at some explanatory text. Can you verify the language we are using, or perhaps suggest something better?

File: web/src/lib/components/pages/record/record.tsx [hunk 1]
```diff
@@ -451,9 +451,9 @@ export default class RecordPage extends Component<RecordProps, RecordState> {
         <span id="why-profile-title">Why a profile?</span>
         <div id="why-profile">
           <p id="why-profile-text">
-            Copy explaining value of profile &amp; demographic capture: Lorem
-            ipsum dolor sit amet, consectetur adipiscing elit. Nulla id orci
-            dui.
+            By providing some information about yourself, the audio data you
+            submit to Common Voice will be more useful to Speech Recognition
+            engines that use this data to improve their accuracy.
```
Comment (kdavis-mozilla) [inline web/src/lib/components/pages/record/record.tsx]: While this is all true and a LGTM is fine, I don't think the "spirit" is captured.

The demographic information provided will help speech recognition engines handle previously overlooked accents, ages, and genders. Effectively allows future engines to better address the full spectrum of spoken language.

Comment (mikehenrty) [reply web/src/lib/components/pages/record/record.tsx]: Yeah, I agree we haven't captured the spirit of Common Voice around its inclusivity. However, I think it's more important for that message to show up front and center on the homepage, which we are currently working on.

Comment (kdavis-mozilla) [reply web/src/lib/components/pages/record/record.tsx]: OK, then LGTM

**Pull request merged**